lib/matplotlib/scale.py

```diff
@@ -79,6 +79,21 @@ def limit_range_for_scale(self, vmin, vmax, minpos):
         return vmin, vmax
 
 
+class CopyableTransformMixin():
```

Comment:
I'll put one last vote in for making this private. It was my impression from the call today that there's no intrinsic reason for `Transform` to not be `__deepcopy__`-able, and so ideally the entire point of this mix-in is as a stand-in for us to be able to label the currently-copy-able Transform subclasses until the more general case gets implemented. 

From that perspective, it seems like adding this class is just creating a public class with the explicit purpose of deprecating it later, when it is unlikely to be very useful for actual users?

But not going to die on this hill, since we need to fix this before 3.4.

Reply:
I'd prefer it be private, but it can't be private because the doc build doesn't work because it tries to reference the private class.  Maybe there is a way to ignore that warning so the docs build?

Reply:
Yeah there are several ways to fix this while leaving it private, let me pull it down real quick so I don't send you down the wrong rabbit hole with preemptive advice.

Reply:
So the doc build fails with 
```
/home/circleci/project/lib/matplotlib/scale.py:docstring of matplotlib.scale.FuncTransform:1: WARNING: py:class reference target not found: matplotlib.scale._CopyableTransformMixin
```
which makes sense because that is a private class...

Reply:
Maybe ?
```
nitpick_ignore = [('py:class', 'matplotlib.scale._CopyableTransformMixin')]
```

Reply:
See https://github.com/astropy/astropy/blob/35501fcba6811705fcd53669742db8346727672d/docs/nitpick-exceptions and https://github.com/astropy/astropy/blob/35501fcba6811705fcd53669742db8346727672d/docs/conf.py#L195

Reply:
That worked - so the mixin is now private again.

Reply:
Sorry for the lag on this. I think the better way here might be to just include `_CopyableTransformMixin` in the documentation, by adding:

```rst
.. automodule:: matplotlib.scale
+   :private-members: _CopyableTransformMixin
   :members:
   :undoc-members:
   :undoc-members:
   :show-inheritance:
```
in `doc/api/scale_api.rst`. Unless you end up moving it into `transforms.py` per the convo above, in which case same change but to `doc/api/transforms_api.rst` should work.

Reply:
That works - it does mean that we now publicly document the private mixin, however (https://51618-1385122-gh.circle-artifacts.com/0/doc/build/html/api/scale_api.html#matplotlib.scale._CopyableTransformMixin), whereas the nitpicky method keeps it undocumented.